onnxruntime/python/tools/transformers/models/t5/convert_to_onnx.py

```diff
@@ -147,10 +154,11 @@ def export_onnx_models(
     disable_auto_mixed_precision: bool = False,
     use_int32_inputs: bool = True,
     model_type: str = "t5",
+    state_dict_path: str = "",
```

Comment:
Shall we make it `None`?

```suggestion
    state_dict_path: Optional[str] = None,
```

Reply:
Dismissing in favor of keeping the API consistent with whisper